### ISSUE TITLE
Add a link to this github repository to the site

### ISSUE THREAD

**Issue by niccokunzmann** [OpenTechSchool/learn.opentechschool.org#6]
When i am on the site and I would like to contribute, I would like to find a link to the possible github repository.

**Comment by Ivoz** [comment]
on `learn.opentechschool.org` there is links to most of the tutorials' source repositories in the "Source" button. What other repositories are you looking for, this one in particular, to be able to make changes to `learn.opentechschool.org` itself?

**Comment by niccokunzmann** [comment]
Yes, I mean a link to this, [OpenTechSchool/learn.opentechschool.org](https://github.com/OpenTechSchool/learn.opentechschool.org), repository.
